.github/workflows/windows.yml

```diff
@@ -483,9 +484,6 @@ jobs:
       INSTALL_DIR: ${{ github.workspace }}/install
       SRC_DIR: ${{ github.workspace }}/src
       BUILD_DIR: ${{ github.workspace }}/build
-      HF_HOME: C:/mount/caches/huggingface
-      OV_CACHE: ${{ github.workspace }}/ov_cache
-      CLEANUP_CACHE: 1
```

Comment:
Just in case - what this variable was supposed to do?

Reply:
HF_HOME - cache for downloaded HF models and datasets, OV_CACHE - internal cache for OV converted models, CLEANUP_CACHE - this option enables automatic cleanup of the test content(models, media, etc) when the tests already used it.

Reply:
Looks like you removed CLEANUP_CACHE: 1 here, is it expected?

Reply:
that is correct, because we will remove the test data by schedule (once per day)